workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/SceneItem.cs

```diff
@@ -6,34 +6,15 @@ namespace Improbable.Gdk.BuildSystem.Configuration
 {
     internal class SceneItem
     {
-        public readonly SceneAsset SceneAsset;
         public bool Included;
-        private readonly bool exists;
+        public readonly bool Exists;
+        public readonly SceneAsset SceneAsset;
 
         public SceneItem(SceneAsset sceneAsset, bool included, SceneAsset[] inAssetDatabase)
         {
             SceneAsset = sceneAsset;
             Included = included;
-            exists = inAssetDatabase.Contains(sceneAsset);
-        }
-
-        public static SceneItem Drawer(Rect position, SceneItem item)
-        {
-            using (item.exists ? null : new GUIColorScope(Color.red))
-            {
-                var positionWidth = position.width;
-                var labelWidth = GUI.skin.toggle.CalcSize(GUIContent.none).x + 5;
-
-                position.width = labelWidth;
-                item.Included = EditorGUI.Toggle(position, item.Included);
-
-                position.x += labelWidth;
-                position.width = positionWidth - labelWidth;
-
-                EditorGUI.ObjectField(position, item.SceneAsset, typeof(SceneAsset), false);
-            }
-
-            return item;
+            Exists = inAssetDatabase.Contains(sceneAsset);
```

Comment:
This field is rather odd, we give it both the array and the sceneAsset, only to check if the array contains it. This can be done before hand?

Reply:
this was also already in before :P
I guess we can move that call to wherever it's needed and remove the field completely

Reply:
Not saying you made a mistake. But if you're simplifying it anyways, might as well fix these ;)